docker-stack.yml

```
@@ -65,7 +66,18 @@ services:
         - com.docker.lb.hosts=${CFGOV_HOSTNAME}
         - com.docker.lb.port=8000
         - com.docker.lb.network=${LB_NETWORK}
-    command: /bin/bash -c './refresh-data.sh && httpd -d cfgov/apache -D FOREGROUND'
+    command: >
+            /bin/bash -c "
+            if django-admin inspectdb wagtailcore_page | grep \"db_table = \'wagtailcore_page\'\" && $REFRESH_DB == false;
+              then 
+                echo 'Skipping DB refresh as db already exists'
+                httpd -d cfgov/apache -D FOREGROUND
+            elif $REFRESH_DB == true;
+              then
+                ./refresh-data.sh && httpd -d cfgov/apache -D FOREGROUND
+            else
+                ./refresh-data.sh && httpd -d cfgov/apache -D FOREGROUND
+            fi "
```

If both elif and else result in the same action, couldn't you just use `else` ?

```suggestion
            else
                ./refresh-data.sh && httpd -d cfgov/apache -D FOREGROUND
            fi "
```

Also, I wonder if using psql could be a simpler test of database existence:

```
psql -l | grep cfgov
```

---

modified the logic a bit to make the conditional more explicit